tldr/api.py

```
@@ -451,24 +451,33 @@ def _get_module_exports(
+            module_file = candidate
+            break
 
-    if not module_file.exists():
+    if module_file is None:
         # Try as directory with __init__.py (Python package)
         init_file = project / module_path / "__init__.py"
         if init_file.exists():
             module_file = init_file
         else:
-            raise ValueError(f"Module not found: {module_path} (tried {module_file} and {init_file})")
+            tried = ", ".join(str(path) for path in checked_paths + [init_file])
```

Ruff (0.15.2)

[warning] Consider `[*checked_paths, init_file]` instead of concatenation

Replace with `[*checked_paths, init_file]`

(RUF005)